{"abstract":[{"code":"URLError","type":"codeVoice"},{"text":" thrown by ","type":"text"},{"type":"codeVoice","code":"URLSession.data(for request: URLRequest)"},{"text":".","type":"text"},{"text":" ","type":"text"},{"text":"A common cause of this error, is due to the user not being connected to the internet.","type":"text"}],"metadata":{"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"urlError","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","text":"URLError","preciseIdentifier":"s:10Foundation8URLErrorV"},{"kind":"text","text":")"}],"externalID":"s:8BrdgeAPI0A12NetworkErrorO03urlD0yAC10Foundation8URLErrorVcACmF","title":"BrdgeNetworkError.urlError(_:)","modules":[{"name":"BrdgeAPI"}],"roleHeading":"Case","symbolKind":"case"},"variants":[{"paths":["\/documentation\/brdgeapi\/brdgenetworkerror\/urlerror(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI","doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI\/BrdgeNetworkError"]]},"kind":"symbol","primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"urlError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"URLError","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation8URLErrorV"},{"text":")","kind":"text"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"minor":3,"major":0,"patch":0},"sections":[],"identifier":{"url":"doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI\/BrdgeNetworkError\/urlError(_:)","interfaceLanguage":"swift"},"references":{"doc://BrdgeAPI/documentation/BrdgeAPI/BrdgeNetworkError":{"type":"topic","abstract":[{"text":"This is an error related to network calls.","type":"text"}],"role":"symbol","navigatorTitle":[{"text":"BrdgeNetworkError","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI\/BrdgeNetworkError","title":"BrdgeNetworkError","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"BrdgeNetworkError","kind":"identifier"}],"url":"\/documentation\/brdgeapi\/brdgenetworkerror"},"doc://BrdgeAPI/documentation/BrdgeAPI":{"identifier":"doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI","type":"topic","title":"BrdgeAPI","url":"\/documentation\/brdgeapi","kind":"symbol","role":"collection","abstract":[{"type":"text","text":"The BR-DGE iOS SDK provides a seamless integration with the BR-DGE APIs through a native interface."}]},"doc://BrdgeAPI/documentation/BrdgeAPI/BrdgeNetworkError/urlError(_:)":{"url":"\/documentation\/brdgeapi\/brdgenetworkerror\/urlerror(_:)","type":"topic","role":"symbol","abstract":[{"code":"URLError","type":"codeVoice"},{"type":"text","text":" thrown by "},{"code":"URLSession.data(for request: URLRequest)","type":"codeVoice"},{"text":".","type":"text"},{"type":"text","text":" "},{"text":"A common cause of this error, is due to the user not being connected to the internet.","type":"text"}],"title":"BrdgeNetworkError.urlError(_:)","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"urlError"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","text":"URLError","preciseIdentifier":"s:10Foundation8URLErrorV"},{"text":")","kind":"text"}],"identifier":"doc:\/\/BrdgeAPI\/documentation\/BrdgeAPI\/BrdgeNetworkError\/urlError(_:)"}}}